sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -229,6 +229,39 @@ abstract class JdbcDialect extends Serializable with Logging{
     }
   }
 
+  /**
+   * Create schema with comment.
+   */
+  def createSchemaWithComment(schema: String, comment: String): Array[String] = {
+    val clauses = ArrayBuilder.make[String]
+    clauses += s"CREATE SCHEMA ${quoteIdentifier(schema)}"
+    if (comment.nonEmpty) clauses += getSchemaCommentQuery(schema, comment)
+    clauses.result()
```

Comment:
why do we return string here? can we pass in `Connection` and do all the work inside this method?

Reply:
similar to `schemasExists`

Reply:
OK